cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -185,6 +185,95 @@ <h2 class="header-slug">
         {% endfor %}
     {% endif %}
 
+
+    {# TODO: Remove the below when input-contains-icon
+             class is added cf-enhancements. #}
+    <div class="block">
+        <div class="block">
+            <div class="input-contains-icon">
+                <div class="input-contains-icon_before
+                            input-contains-icon_before__search">
+                    <span class="input-contains-icon_icon"></span>
```

Comment:
Updated in https://github.com/cfpb/cfgov-refresh/commit/91fbae6147b56362888b13d8c47d35d5d3284761